src/server/utils/jsUtils.js

```diff
@@ -197,10 +197,20 @@ function retryAsync(promiseFunc, maxTries, delay, iteration) {
     });
 }
 
+function stripErrorColon(err) {
```

Comment:
You're right! I don't know what I commented this, I confused logging with throwing an error, I read above that we log an error with 'Error' but it is a log, not throwing an error. Just a problem with my brain, forget this :)